src/ResourceManager/ApiManagement/Commands.ApiManagement.ServiceManagement/Commands/NewAzureApiManagementLogger.cs

```diff
@@ -38,26 +41,37 @@ public class NewAzureApiManagementLogger : AzureApiManagementCmdletBase
         public String LoggerId { get; set; }
 
         [Parameter(
+            ParameterSetName = EventHubLoggerSet,
             ValueFromPipelineByPropertyName = true,
             Mandatory = true,
             HelpMessage = "EventHub Entity name. This parameter is required.")]
         [ValidateNotNullOrEmpty]
         public String Name { get; set; }
 
         [Parameter(
+            ParameterSetName = EventHubLoggerSet,
             ValueFromPipelineByPropertyName = true,
             Mandatory = true,
-            HelpMessage = "EventHub Connection String with Send Policy Rights. This parameter is required.")]
+            HelpMessage = "EventHub Connection String with Send Policy Rights. This parameter is optional.")]
```

Comment:
@solankisamir "This parameter is optional" even though `Mandatory = true`

Reply:
will fix it.